android/src/main/java/co/nimblehq/kaylabruce/kmmic/android/presentation/screen/surveydetail/SurveyDetailScreen.kt

```diff
@@ -147,5 +159,8 @@ private fun StartButton() {
 @Preview
 @Composable
 fun PreviewSurveyDetailScreen() {
-    SurveyDetailScreen()
+    SurveyDetailScreen(
+        onNavigateToQuestion = { id, number -> },
```

Comment:
In case we don't use the parameter, we could rename it to `_` as IDE's suggestion
```suggestion
        onNavigateToQuestion = { _, _ -> },
```

<img width="400" alt="Screenshot 2023-09-20 at 1 45 53 PM" src="https://github.com/nimblehq/ic-kmm-kayla-bruce/assets/25218255/c7ae4cff-f221-4c38-8435-c2d84e591c4a">